src/pudl/convert/censusdp1tract_to_sqlite.py

```diff
@@ -48,18 +50,13 @@ def censusdp1tract_to_sqlite(pudl_settings=None, year=2010):
         None
 
     """
-    if pudl_settings is None:
```

Comment:
Rather than calling `Datastore()` in the function call signature it might be cleaner to do have a default value of `None` and then:

```py
if ds is None:
    ds = Datastore()
```

This avoids `Datastore()` getting called when the function is defined (I think?) and prevents that input from being a persistent and potentially mutable value outside the scope of the function.

I know we've got lots of example of this floating around that we should probably clean up.

Reply:
Updated 👍